src/Test/L0/Worker/JobRunnerL0.cs

```diff
@@ -1,12 +1,17 @@
+using Microsoft.TeamFoundation.DistributedTask.WebApi;
 using Microsoft.VisualStudio.Services.Agent.Worker;
 using System;
+using System.Collections.Generic;
 using Xunit;
 
 namespace Microsoft.VisualStudio.Services.Agent.Tests.Worker
 {
     public class JobRunnerL0
     {
         //// TODO: Fix this test. This test hit file in use issues trying to write to the log file. My guess is it probably has to do with XUnit running tests in parallel.
+        // [Fact]
+        // [Trait("Level", "L0")]
+        // [Trait("Category", "JobRunnerL0")]
```

Comment:
I know this change looks strange. I partially fixed the commented out code :)

I was using this file to test ad-hoc things (e.g. JsonUtility.ToString does not null ref).